addon/templates/components/polaris-text-field.hbs

```diff
@@ -0,0 +1,118 @@
+<!-- template-lint invalid-interactive=false -->
```

Comment:
Can this go right above the line that causes the linter to get angry?

Reply:
ESLint lets you specify to ignore a rule just on the next line - if `template-lint` will let you do that, I'd prefer it 👍